app/submissions/templates/submissions/base.html

```diff
@@ -0,0 +1,88 @@
+{% load static %}
```

Comment:
I'm assuming again this is just a shortcut to get things going? There's a lot of code here that can be shared between registries and submissions.